tests/unit/connector/test_http_input.py

```diff
@@ -38,6 +63,8 @@ def setup_method(self):
             "/(first|second)/jsonl": "jsonl",
             "/(third|fourth)/jsonl*": "jsonl",
             "/plaintext": "plaintext",
+            "/auth-json-secret": "json",
+            "/auth-json-file": "json",
         },
```

Comment:
```suggestion
            "/.*/[A-Z]{2}/json$": "json",
```

* insert regex endpoint for usage below